testdata/go/v4-alpha/monitoring/memcached-operator/test/e2e/e2e_test.go

```diff
@@ -237,7 +237,12 @@ var _ = Describe("memcached", Ordered, func() {
 
 			By(fmt.Sprintf("validating MemcachedDeploymentSizeUndesiredCountTotal has increased by %d", numberOfScales))
 			finalMetricValue := getMetricValue(memcachedDeploymentSizeUndesiredCountTotalName)
-			Expect(finalMetricValue).To(Equal(initialMetricValue + numberOfScales))
+
+			isFinalValueGreater := false
+			if finalMetricValue >= initialMetricValue+numberOfScales {
+				isFinalValueGreater = true
+			}
+			Expect(isFinalValueGreater).To(BeTrue())
```

Comment:
Since the tests use `gomega`,  I *think* this could be simplified to one line with:
```go
Expect(1.0).Should(BeNumerically(">=", 1.0))
```
For more info on the `BeNumerically` function you can take a look at the [`gomega` go docs entry](https://pkg.go.dev/github.com/onsi/gomega#BeNumerically)

Reply:
Done